doc/src/explanation/active-deprecations.md

```diff
@@ -1031,7 +1031,7 @@ just printing support then perhaps `TableForm` can be used.
 
 It isn't clear what to advise as a replacement here without knowing more about
 the usecase. If you are unclear how to update your code, please [open an
-issue](https://github.com/sympy/sympy/issues/new) or [write to our mailing
+issue](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2Fsympy%2Fsympy%2Fissues%2Fnew) or [write to our mailing
```

Comment:
Does this need to be fixed?

Reply:
They both work, I supposed that since the linkcheck command suggested to redirect it it was the best thing to do, but since they work we can do what we want with them.
For now, in the last commit, I put this one [https://github.com/sympy/sympy/issues/new](url). Is it ok? What do you think?

Reply:
The link checker will indicate that it is redirecting, but it is a human decision as to whether it should be fixed. This is only a redirect if we are not logged in to github, so we should not fix it.

Reply:
Ok, I'll note that down, didn't quite know how the link checker works so I just assumed the redirects were all to be updated, the next time I'll work on something like this I'll be more careful and precise.
Anyway, I've already changed the link to the previous one, so it should be already taken care of.
Thank you for your patience and explanations, really appreciate it ;)

Reply:
I asked about fixing `sympy/simplify/ratsimp.py`. What are your plans for this?

Reply:
Hi there!
I think that I don't really know what you are referring to, is the `ratsimp.py` fix related to the redirected links issue? If yes, in which way?